packages/styling/src/styles/theme.ts

```diff
@@ -108,21 +108,30 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
-  const typography = { ...DefaultTypography, ...theme.typography };
+  let typography = { ...DefaultTypography };
+  if (theme.typography) {
```

Comment:
these spreads have real run-time cost/impact. We should not even do this spread if `theme.typography` is true (since you wont' be using this at all).